pmc/challenges/indexing-querying.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "indexing-querying",
+  "description": "Indexes half the data for NYC Taxis, then concurrently queries Elasticsearch while indexing again.",
+  "default": false,
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "put-settings",
+        "body": {
+          "transient": {
+            "search.default_search_timeout": "{{default_search_timeout | default(-1)}}"
+          }
+        }
+      }
+    },
+    {
+      "operation": "delete-index"
+    },
+    {
+      "operation": {
+        "operation-type": "create-index",
+        "settings": {{index_settings | default({}) | tojson}}
+      }
+    },
+    {
+      "name": "check-cluster-health",
+      "operation": {
+        "operation-type": "cluster-health",
+        "index": "pmc",
+        "request-params": {
+          "wait_for_status": "{{cluster_health | default('green')}}",
+          "wait_for_no_relocating_shards": "true"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "name": "initial-index-append",
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": {{bulk_size | default(500)}},
+        "ingest-percentage": 50
+      },
+      "warmup-time-period": 30,
+      "clients": {{bulk_indexing_clients | default(8)}},
+      "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+    },
+    {
+      "name": "refresh-after-index",
+      "operation": "refresh"
+    },
+    {
+      "operation": {
+        "operation-type": "force-merge",
+        "request-timeout": 7200
+      }
+    },
+    {
+      "name": "refresh-after-force-merge",
+      "operation": "refresh"
+    },
+    {
+      "name": "wait-until-merges-finish",
+      "operation": {
+        "operation-type": "index-stats",
+        "index": "_all",
+        "condition": {
+          "path": "_all.total.merges.current",
+          "expected-value": 0
+        },
+        "retry-until-success": true,
+        "include-in-reporting": false
+      }
+    },
+    {
+      "parallel": {
+        "tasks": [
+          {
+            "name": "concurrent-index-append",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": {{bulk_size | default(500)}},
+              "ingest-percentage": 50
```

Comment:
do we only want to index half of the data?

Reply:
I think this makes sense, we ingest the first half here:
https://github.com/elastic/rally-tracks/blob/ee2a1f06940b62f34a696a6a0bebf32e7e25993d/nyc_taxis/challenges/indexing-querying.json#L32-L37

Then during querying, we ingest the remaining 50%:
https://github.com/elastic/rally-tracks/blob/ee2a1f06940b62f34a696a6a0bebf32e7e25993d/nyc_taxis/challenges/indexing-querying.json#L61-L82